xhprof_lib/config.sample.php

```diff
@@ -11,6 +11,7 @@
 $_xhprof['servername'] = 'myserver';
 $_xhprof['namespace'] = 'myapp';
 $_xhprof['url'] = 'http://url/to/xhprof/xhprof_html';
+$_xhprof['getparam'] = "_profile";
```

Comment:
Well yeah I haven't thought about that, but to keep BC Compat I could also add a check to verify if the var is set or not, and if not will use the default